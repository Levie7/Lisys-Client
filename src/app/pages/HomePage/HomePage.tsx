import React from 'react';

export const HomePage = () => {
    return (
        <div className='HomePage' style={{ margin: '24px 16px 0' }}>
            <header className='HomePage-header'>Home Page</header>
        </div>
    );
};
