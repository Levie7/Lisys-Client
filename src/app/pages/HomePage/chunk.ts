import { PageLoading } from 'src/app/shell/PageLoading';

import { suspend } from 'src/core/chunk';

export const HomePage = suspend({
    component: () => import(/* webpackChunkName: "HomePage" */ './index'),
    renderWhileLoading: PageLoading,
});
