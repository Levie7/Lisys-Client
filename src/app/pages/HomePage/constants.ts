export const notification = {
    description: {
        en: 'You have successfully logged in to Lisys!',
        id: 'Kamu berhasil masuk di Lisys!',
    },
    title: { en: 'Welcome', id: 'Selamat Datang' },
};
