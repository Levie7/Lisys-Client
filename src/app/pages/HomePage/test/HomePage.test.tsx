import { shallow } from 'enzyme';
import * as React from 'react';

import { HomePage } from '../HomePage';

describe('HomePage', () => {
    const wrap = shallow(<HomePage />);

    it('<HomePage/> - should render HomePage component correctly', () => {
        expect(wrap).toMatchSnapshot();
    });
});
