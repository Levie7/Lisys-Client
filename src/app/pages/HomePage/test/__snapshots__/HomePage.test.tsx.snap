// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage <HomePage/> - should render HomePage component correctly 1`] = `ShallowWrapper {}`;
